web/crux/prisma/schema.prisma

```diff
@@ -47,20 +47,21 @@ model UsersOnTeams {
 }
 
 model Node {
-  id             String    @id @default(uuid()) @db.Uuid
-  createdAt      DateTime  @default(now()) @db.Timestamptz(6)
-  createdBy      String    @db.Uuid
-  updatedAt      DateTime? @default(now()) @db.Timestamptz(6)
-  updatedBy      String?   @db.Uuid
+  id             String        @id @default(uuid()) @db.Uuid
+  createdAt      DateTime      @default(now()) @db.Timestamptz(6)
+  createdBy      String        @db.Uuid
+  updatedAt      DateTime?     @default(now()) @db.Timestamptz(6)
+  updatedBy      String?       @db.Uuid
   name           String
+  type           NodeTypeEnum? @default(dagent)
```

Comment:
Do we have to add a default?

Reply:
guess it's necessary for the auto migration.

Reply:
however we can copy the manual inserts from the 50th migration and do something like that

Reply:
It does not hurt, while for compatibility it is small win. Environments that were set up earlier will be interoperable, because they are all docker typed.